File: DaS.ScriptLib.CS/Game/Mem/GameStats.vb
Imports DaS.ScriptLib.Injection.Structures

Namespace Game.Mem

    Public Class GameStats

        Public Shared ClearCount As New LivePtrVar(Of Int32)(Function() Ptr.GameStatsPtr.Value + &H3C)
        Public Shared TrueDeathCount As New LivePtrVar(Of Int32)(Function() Ptr.GameStatsPtr.Value + &H58)
        Public Shared TotalPlayTime As New LivePtrVar(Of Int32)(Function() Ptr.GameStatsPtr.Value + &H68)

    End Class

End Namespace

File: DaS.ScriptLib.CS/Game/Data/Structures/GameStats.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
using System.Threading.Tasks;
using DaS.ScriptLib.Injection.Structures;

namespace DaS.ScriptLib.Game.Mem
{
	public class GameStats
	{
		public static LivePtrVar<Int32> ClearCount = new LivePtrVar<Int32>(() => Ptr.GameStatsPtr.Value + 0x3c);
		public static LivePtrVar<Int32> TrueDeathCount = new LivePtrVar<Int32>(() => Ptr.GameStatsPtr.Value + 0x58);
		public static LivePtrVar<Int32> TotalPlayTime = new LivePtrVar<Int32>(() => Ptr.GameStatsPtr.Value + 0x68);
	}
}
